src/sentry/migrations/0203_use_project_team_for_team_key_transactions.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 1.11.29 on 2021-06-07 14:21
+
+import django.db.models.deletion
+from django.db import migrations
+
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0202_org_slug_upper_idx"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="teamkeytransaction",
+            name="project_team",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                db_constraint=False,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="sentry.ProjectTeam",
+            ),
+        ),
+        migrations.AlterUniqueTogether(
+            name="teamkeytransaction",
+            unique_together={("project_team", "transaction")},
+        ),
+        migrations.RemoveField(
+            model_name="teamkeytransaction",
+            name="project",
+        ),
+        migrations.RemoveField(
+            model_name="teamkeytransaction",
+            name="team",
+        ),
```

Comment:
Does this table have any reads on it currently?

If so, you'll need to separate the state and schema here. Removing columns from models will cause 500s if django does any queries to them, as migrations go out way before the code does.

Reply:
All the reads are currently behind a disabled feature flag at the moment.